invenio_records_rest/serializers/marshmallow.py

```diff
@@ -26,28 +26,35 @@
 
 from __future__ import absolute_import, print_function
 
-from .base import PreprocessorMixin
+from .base import TransformerMixinInterface
+from .schemas.json import RecordSchemaJSONV1
 
 
-class MarshmallowSerializer(PreprocessorMixin):
+class MarshmallowMixin(TransformerMixinInterface):
     """Base class for marshmallow serializers."""
 
-    def __init__(self, schema_class=None, replace_refs=False):
+    def __init__(self, schema_class=RecordSchemaJSONV1, **kwargs):
```

Comment:
@lnielsen I set the default schema as RecordSchemaJSONV1. It is equivalent to not giving a schema with `PreprocessorMixin`. Only `revision` is filtered out.